linalg/hypre.cpp

```diff
@@ -3573,6 +3573,33 @@ void HypreSmoother::SetOperator(const Operator &op)
 #endif
    }
 
+#if MFEM_HYPRE_VERSION < 22100
+   switch (type)
+   {
+      case 3:
+      case 6:
+      case 8:
+      case 10:
+      case 13:
+      case 14:
+         Z = new HypreParVector(*A);
+   }
+#elif defined(HYPRE_USING_GPU) || defined(HYPRE_USING_CUDA) || defined(HYPRE_USING_HIP)
```

Comment:
This should be equivalent:
```suggestion
#elif defined(HYPRE_USING_GPU)
```

Reply:
Done 👍